versions.yaml

```diff
@@ -249,7 +249,7 @@ externals:
     uscan-url: >-
       https://github.com/opencontainers/runc/tags
       .*/v?(\d\S+)\.tar\.gz
-    version: "v1.1.0"
+    version: "v1.1.4"
```

Comment:
@GabyCT Could you include a summary of what changed in the latest version in the commit message.

Reply:
@amshinde changes applied